certbot/storage.py

```diff
@@ -256,8 +259,26 @@ def __init__(self, config_filename, cli_config):
         self.live_dir = os.path.dirname(self.cert)
 
         self._fix_symlinks()
+        if update_symlinks:
+            self._update_symlinks()
         self._check_symlinks()
 
+    def __str__(self):
+        """Returns a string representation of the object."""
+        # import ipdb; ipdb.set_trace()
+        outdict = {"Lineage": self.lineagename,
+                   "Domains": " ".join(self.names())}
+        return "RenewableCert(%s)" % str(outdict)
```

Comment:
I think the most UNIX-y output data would be a table that people can parse with `cut` and/or `awk` by default.